RecoTracker/SpecialSeedGenerators/python/SimpleCosmicBONSeeder_cfi.py

```diff
@@ -51,7 +51,7 @@ def makeSimpleCosmicSeedLayers(*layers):
         originRadius     = cms.double(150.0),  #  |-> probably don't change
         originHalfLength = cms.double(90.0),   # /    anything at all.
         ptMin = cms.double(0.5),               # pt cut, applied both at the triplet finding and at the seeding level
-        pMin  = cms.double(1.0),               # p  cut, applied only at the seeding level
+        pMin  = cms.double(4.0),               # p  cut, applied only at the seeding level
```

Comment:
this cut is applied at the seed level, while the one in the alignment producer is applied after full track building and fitting. Presumably the two do not necessarily coincide. Have you check what's the price in efficiency loos for ALCARECO tracks this cut implies?
Also, this is changing reconstruction for every potential consumer, not just for alignment.
Instead of increasing the threshold, couldn't a different strategy be envisaged to clean the noise clusters, be it in h/w or s/w?